package.json

```diff
@@ -18,20 +18,26 @@
   "author": "jimmay5469",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.2",
-    "ember-cli": "0.2.3",
-    "ember-cli-app-version": "0.3.3",
-    "ember-cli-content-security-policy": "0.4.0",
```

Comment:
Revert, this gets removed by the default blueprint now, but we should keep it, along with a proper CSP config for the dummy app.